examples/python/RVC4/run_gpustereo_example.py

```
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Ruff (0.15.12)

[warning] Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)